### PR TITLE
build(deps): Update dependencies in docs project

### PR DESCRIPTION
**Pull request by tylerbutler**
Updates the dependencies in the docs project to the latest versions.

### CODE REVIEW

File: docs/.nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-14
```
Comment (Josmithr) [inline docs/.nvmrc:1]: Do we need this file here? Shouldn't we just have a single nvmrc at the repo root?

**Review by Josmithr — APPROVED** [review]
🚀

File: docs/.nvmrc (hunk 1 above)
Comment (tylerbutler) [reply docs/.nvmrc:1]: Each release group/independent package has its own.

Comment (tylerbutler) [reply docs/.nvmrc:1]: This lets us have different settings per release group, which can be useful for settings like peer dependency strictness.

Comment (tylerbutler) [reply docs/.nvmrc:1]: Oops, I thought this was referring to a different file. You're right that we can delete this.

**Pull request merged**